quickcomm/tests/test_views.py

```diff
@@ -34,10 +35,51 @@ def test_logout(self):
             'password': 'pass',
         })
 
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, 302)
         self.assertTrue(
             response.wsgi_request.user.is_authenticated)
 
         response = self.client.post('/logout/', {})
         self.assertEqual(response.status_code, 302)
         self.assertFalse(response.wsgi_request.user.is_authenticated)
+        
+class EditProfileViewTest(TestCase):
+    def setUp(self):
+        
+        user = User.objects.create_user(
+            username='user',
+            password='pass'
+        )
+        
+        user.save()
+        
+        author = Author.objects.create(user=user, host='http://127.0.0.1:8000', display_name='First Name', github='https://github.com/', profile_image='https://www.history.com/.image/c_fit%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_620/MTU3ODc5MDg2NDM2NjU2NDU3/reagan_flags.jpg')
+        author.full_clean()
+        
+        
+    def test_edit_not_logged_in(self):
+        pass
+    
+    
+    def test_edit_name(self):
+        # c = Client()
+        # author = Author.objects.all()[0]
+        
+        # c.post('/login/', {
+        #     'display_name': 'user',
```

Comment:
Why is there a huge commented out section not tested?

Again, can probably pass for the time being.

Reply:
I couldn't get the test to work properly, so I resolved to leave it for now and come back later.